dockerfiles/Dockerfile.ecs

```diff
@@ -0,0 +1,2 @@
+FROM ghcr.io/fluent/fluent-bit/master:x86_64
```

Comment:
This will only work for integration tests off `master` I think.
We need to be able to trigger them on a PR.